SPEC.md

```diff
@@ -106,6 +106,11 @@ This <ins>optional</ins> key’s value (type string) shall contain the PCI addre
 of the associated physical interface (in standard BDF format “dddd:BB:DD.f”) if
 available and applicable.
 
+#### 3.1.3.5 “representor-device”
+
+This <ins>optional</ins> key’s value (type string) shall contain the name
+of the representor device for PCI VF (e.g. "eth3") if available and applicable.
+
 ### 3.1.4 “vdpa”
```

Comment:
vdpa devices can also have representor devices. Can you please add the key to it's section as well?

Reply:
done